Python/Product/TestAdapter.Executor/TestExecutor.cs

```diff
@@ -573,6 +605,12 @@ out testMethod
                             }
                         }
                     }
+                    if (File.Exists(testList)) {
+                        try {
```

Comment:
It's deliberate. If tests fail, I want to be able to find the file later on. It's in TEMP, so it'll be cleaned up eventually anyway.